papers/MCMC/Surveys/DSA.ecsv

```diff
@@ -0,0 +1,50 @@
+# %ECSV 1.0
```

Comment:
should this be in `zdm/data/Surveys` ?

[same for FAST below]

Reply:
@JordanHoffmann3 ? I tend to agree with X. If this is for a previous DSA sample - e.g., one with only a few FRBs - then maybe we create a new directory under "Surveys" titled "Historical"? But then again, if this is now a deprecated version of DSA that contained only some of their FRBs I think it's OK to stay here